api/candlepin-api-spec.yaml

```diff
@@ -8236,7 +8238,7 @@ components:
               type: string
               example: "/jobs/ff808081554a3e4101554a3e9033005d"
             resultData:
-              type: string
+              type: object
```

Comment:
Updated to object type as some of the async jobs return objects instead of a string in the resultData field and this causes deserialization issues with the OpenApi apis.